src/core/pile.sol

```diff
@@ -128,11 +128,20 @@ contract Pile is DSNote, TitleOwned {
         beans.drip(loans_[loan].fee);
```

Comment:
I think we can think about taking this function abstraction out if it just calls only `beans.drip`

Reply:
Currently, it is easier because bean doesn't know the `fee` of a loan.

You would need to ask pile about the fee first